spec/spec_join_addfields/section/join-vocabulary.md

```diff
@@ -0,0 +1,116 @@
+| `rml:addFieldAlias`       | `rml:Join` | `rml:FieldAlias`    | 
+
+<aside class="issue">
+Els: in the RML CORE spec rml:Join is the class containing the join conditions. Can this still be adapted?
+</aside>
+
+### Parent Logical Source
+
+The Range of a <dfn>parent logical source</dfn> property is a [Logical Source](https://kg-construct.github.io/rml-io/spec/docs/#defining-logical-sources) and describes the data source that supplies the additional data and is the right side of the join. 
+
+### Join Conditions
+
+A <dfn>Join Condition</dfn> is represented by a resource that MUST contain exactly one value for each of the following two properties:
+
+- a <dfn>child map</dfn> (`rml:childMap`), whose value is an [Expression Map](https://kg-construct.github.io/rml-core/spec/docs/#expression-map-rml-expressionmap) (`rml:ExpressionMap`), 
+which MUST include references that exists in the child logical source, or it should have a constant value.
+
+- a <dfn>parent map</dfn> (`rml:parentMap`), whose value is an [Expression Map](https://kg-construct.github.io/rml-core/spec/docs/#expression-map-rml-expressionmap) (`rml:ExpressionMap`),
+which, as the join condition's parent map, MUST include references that exists in the logical source specified by the [=parent logical source=] property or it should have a constant value.
+
+
+| Property                    | Domain               | Range                     |
+| --------------------------- | -------------------- | ------------------------- |
+| `rml:childMap`              | `rml:JoinCondition`  | `rml:ExpressionMap`       |
+| `rml:parentMap`             | `rml:JoinCondition`  | `rml:ExpressionMap`       |
+
+#### Shortcuts
+
+If the value of the [=child map=] property (`rml:childMap`) is a [reference-valued Expression Map](https://kg-construct.github.io/rml-core/spec/docs/#reference-rml-reference),
+then the `rml:child` shortcut could be used.
+
+Similarly, if value of the [=parent map=] (`rml:parentMap`) is a [reference-valued Expression Map](https://kg-construct.github.io/rml-core/spec/docs/#reference-rml-reference),
+then the `rml:parent` shortcut could be used.
+
+| Property                    | Domain               | Range                     |
+| --------------------------- | -------------------- | ------------------------- |
+| `rml:child`                 | `rml:JoinCondition`  | `Literal`                 |
+| `rml:parent`                | `rml:JoinCondition`  | `Literal`                 |
+
+
+<aside class="note">
+This definition should be in line with the definition in RML CORE, with one small difference: it refers directly to a parent logicial source, and not to the logical source of the parent triples map.
+</aside>
+<aside class="issue">
+Els: or can we just refer to rml core and not specify any definitions here?
+</aside>
+<aside class="issue">
+Els: can we also optionally declare a join function here, to allow not only equijoins (default) but also other joins
+</aside>
+
+### Add Field 
```

Comment:
TODO clarify that the reference expression of the parent logical source must then be used, and overwrites any conflicting reference expressions of the child logical source? (so you can then refer to the Alias)